specs/AdaptiveCardsXamarinSpec.md

```diff
@@ -0,0 +1,188 @@
+| Rendering | ```(ACRRenderResult*) render:(ACOAdaptiveCard*) config:(ACOHostConfig) widthConstraint:(float) delegate:(id<ACRActionDelegate>)``` | P0 |
+| Rendering | ```(ACRRenderResult*) renderAsViewController:(ACOAdaptiveCard*) config:(ACOHostConfig) frame:(CGRect) delegate:(id<ACRActionDelegate>)``` | P0 |
+| Custom Element | ```(void) setBaseCardElementRenderer: (ACRBaseCardElementRenderer*)cardElementType: (ACRCardElementType)``` | P1 |
+| Custom Element | ```(void) setCustomElementParser: (NSObject<ACOIBaseCardElementParser>)``` | P1 |
+| Custom Element | ```(void) setCustomElementParser: (NSObject<ACOIBaseCardElementParser>) key: (NSString*)``` | P1 |
+| Custom Element | ```(void) setCustomElementRenderer: (ACRBaseCardElementRenderer*) key:(NSString *)``` | P1 |
+| Custom Element | ```(NSObject<ACOIBaseCardElementParser>*) getCustomElementParser: (NSString*)``` | P1 |
+| Custom Element | ```(ACRBaseCardElementRenderer*) getRenderer: (NSNumber *)``` | P1 |
+| Custom Action | ```(void) setActionRenderer: (ACRBaseActionElementRenderer) cardElementType: (NSNumber*)``` | P1 |
+| Custom Action | ```(ACRBaseActionElementRenderer*) getActionRenderer: (NSNumber*)``` | P1 |
+| Custom Action Set | ```(void) setActionSetRenderer: (id<ACRIBaseActionSetRenderer>)``` | P2 |
+| Custom Action Set | ```(id<ACRIBaseActionSetRenderer>) getActionSetRenderer``` | P2 | 
+| Resource Resolvers | ```(void) setResourceResolver: (NSObject<ACOIResourceResolver>) scheme: (NSString*)``` | P2 |
+| Resource Resolvers | ```(NSObject<ACOIResourceResolver>) getResourceResolverForScheme: (NSString*)``` | P2 |
+| Feature Registration | ```(void) addFeature: (NSString*) featureVersion: (NSString*)``` | P0 | 
+| Feature Registration | ```(NSString*) removeFeature: (NSString*)``` | P0 |
+| Feature Registration | ```(NSString*) getFeatureVersion: (NSString*)``` | P0 |
+| Resource Information | ```(NSArray<ACORemoteResourceInformation>*) remoteResourceInformation``` | P2 | 
+
+## Breaking changes
+
+There is one big breaking change to be introduced in the iOS platform due to the Xamarin bindings having a direct relationship with the name of the library, which requires changing the library name from ACFramework to AdaptiveCards.
+
+Besides that, no breaking changes should be introduced as the native platforms' code is only consumed and projected into C#.
+
+## Test Plan
+
+For the alpha and pre-release versions the testing will be performed manually
+
+## Release plan
+
+### Artifacts
+
+The artifacts to be released with this library will be:
+1. Source code for Xamarin bindings in the github repository
+1. Xamarin.Android nuget package
+1. Xamarin.iOS nuget package
+
+### Continuous Integration and Release
+
+To be able to release the previously mentioned nuget packages build definitions must be created. The current system we use for building the other packages allow us to build Xamarin.Android and Xamarin.iOS packages
```

Comment:
>urrent system we u [](start = 102, length = 18)

Nit: We should refer to this as ADO (Azure Deployment Pipeline) or whatever the new name is now - @paulcam206